website/docs/compute/cloud-deployment/configure-storage.md

```diff
@@ -0,0 +1,146 @@
+# Configure Storage
+
+By default, Compute will store compiled functions and function source code on the local storage in your Kubernetes cluster. For greater scalability, Compute can be configured to store artifacts in cloud-based object storage like Amazon S3 or Google Cloud Storage.
+
+## Kubernetes deployment
```

Comment:
Remove this? 

It only show part of the spec and then the spec with the secret is described below. Either way we shouldn't split the k8s deployment section with docker compose in the middle.

Reply:
Hrm. I feel like it's kind of important to have this section since this is the only "officially" supported deployment configuration. The docker-compose version is really only meant for local development.